### PR TITLE
support managed identity

### PR DESCRIPTION
**Pull request by necusjz**
generate commands to support identity according to https://github.com/Azure/azure-cli/blob/20ffb2d35d59a104641d75aeb76b6f31885e26aa/doc/managed_identity_command_guideline.md

main command:
image

subcommand:
image

related runtime check-in: https://github.com/Azure/azure-cli/pull/29953

### CODE REVIEW

File: src/aaz_dev/command/model/configuration/_schema.py [hunk 1]
```diff
@@ -139,6 +139,8 @@ class CMDSchemaBase(Model):
     default = ModelType(CMDSchemaDefault)
     nullable = CMDBooleanField()  # whether null value is supported
 
+    action = StringType()  # distinguish identity assign and remove
+
```
Comment (kairu-ms) [inline src/aaz_dev/command/model/configuration/_schema.py:143]: I think the action should be defined in  CMDIdentityObjectSchemaBase

**Comment by kairu-ms** [comment]
@necusjz could you create a draft PR in aaz to show the model change? Thanks. I want to review the change in model first. Thanks.

File: src/aaz_dev/command/model/configuration/_schema.py (hunk 1 above)
Comment (necusjz) [reply src/aaz_dev/command/model/configuration/_schema.py:143]: after rethinking, in subcommand `action` is used by **AAZListType** and **AAZStrType**:
```python
def _update_instance(self, instance):
    _instance_value, _builder = self.new_content_builder(
        self.ctx.args,
        value=instance,
        typ=AAZIdentityObjectType
    )
    _builder.set_prop("userAssigned", AAZListType, ".mi_user_assigned", typ_kwargs={"flags": {"action": "assign"}})
    _builder.set_prop("systemAssigned", AAZStrType, ".mi_system_assigned", typ_kwargs={"flags": {"action": "assign"}})

    user_assigned = _builder.get(".userAssigned")
    if user_assigned is not None:
        user_assigned.set_elements(AAZStrType, ".")

    return _instance_value
```

File: src/aaz_dev/command/model/configuration/_instance_update.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 from schematics.models import Model
-from schematics.types import ModelType
+from schematics.types import ModelType, StringType
```
Comment (kairu-ms) [inline src/aaz_dev/command/model/configuration/_instance_update.py:6]: Why add `StringType` here?

File: src/aaz_dev/command/model/configuration/_schema.py (hunk 1 above)
Comment (kairu-ms) [reply src/aaz_dev/command/model/configuration/_schema.py:143]: The flags can added in the identityobject level right? 
```
_instance_value, _builder = self.new_content_builder(
        self.ctx.args,
        value=instance,
        typ=AAZIdentityObjectType,
        typ_kwargs={"flags": {"action": "assign"}}
    )
```

Comment (necusjz) [reply src/aaz_dev/command/model/configuration/_schema.py:143]: i remember it has single instance logic

Comment (kairu-ms) [reply src/aaz_dev/command/model/configuration/_schema.py:143]: what is single instance logic

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**